app/views/UnauthorisedView.scala.html

```diff
@@ -27,5 +27,9 @@
 
     <h1 class="govuk-heading-xl">@messages("unauthorised.heading")</h1>
 
-    <p class="govuk-body">@messages("unauthorised.guidance")</p>
+              <p class="govuk-body">@messages("unauthorised.paragraph")
+                    <a href="https://www.gov.uk/government/organisations/hm-revenue-customs/contact/pension-scheme-enquiries"
```

Comment:
This shouldn't be hard-coded on the template. As a reference, please have a look at `ReturnSubmittedController`. Happy to discuss if needed.